test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -1890,14 +1907,14 @@ describe('the rubicon adapter', function () {
           );
 
           const [request] = spec.buildRequests(bidderRequest.bids, bidderRequest);
-          expect(request.data.imp[0].ext.context.data.pbadslot).to.equal('1234567890');
+          expect(request.data.site.ext.data.pbadslot).to.equal('1234567890');
         });
 
         it('should include GAM ad unit in bid request', function () {
           createVideoBidderRequest();
           bidderRequest.bids[0].fpd = {
             context: {
-              adserver: {
+              adServer: {
```

Comment:
Was this a previous bug being fixed?

Were we looking at all lowercase when it should have been camelCase the whole time?

Reply:
Yes this should have been camelCase